tox.ini

```diff
@@ -108,7 +108,7 @@ extras = {[testenv:docs-no-checks]extras}
 deps =
     py39-sphinx61-docs: sphinx~=6.1.0
 commands =
-    sphinx-build -b html docs/ docs/_build/html -v -w warnings.txt {posargs}
+    sphinx-build -b html docs/ docs/_build/html -nTv -w warnings.txt {posargs}
```

Comment:
`-n` forces nitpicky mode. Not strictly necessary, since we set `nitpicky=True` in `conf.py` now

`-T` show full traceback on error.